SUBMODULE_VERSIONS

```diff
@@ -4,6 +4,7 @@ daff5fead3fbe22c6fc58310ca3f49caf117f185 third_party/benchmark
 a5d9d0f7d368054fd1691aedf1db4116efcc233e third_party/flatbuffers
 f2fb48c3b3d79a75a88a99fba6576b25d42ec528 third_party/googletest
 755a89591528b692315ad0325347e2fd4637271b third_party/llvm-project
+17b12a4481daa150e2d1ea3ada086b551b856707 third_party/marl
```

Comment:
Technically adding the marl submodule doesn't need to happen at the same time as these other changes, but I'm fine with adding it in this PR.